bin/test-create-block.sh

```diff
@@ -20,14 +20,47 @@ error () {
 }
 
 cleanup() {
-	rm -rf "$DIRECTORY/esnext-test"
+	rm -rf "$DIRECTORY/example-static-es5"
+	rm -rf "$DIRECTORY/example-static"
 }
 
 trap cleanup EXIT
 
-status "Scaffolding block..."
-npx wp-create-block esnext-test --no-wp-scripts
-cd esnext-test
+# First test block
+
+status "Scaffolding Example Static (ES5) block..."
+npx wp-create-block example-static-es5 -t es5
+cd example-static-es5
+
+status "Verifying project..."
+expected=8
+actual=$( find . -maxdepth 1 -type f | wc -l )
+if [ "$expected" -ne "$actual" ]; then
+	error "Expected $expected files in the project root, but found $actual."
+    exit 1
+fi
+
+cd ..
+
+# Second test block
+
+status "Scaffolding Example Static block..."
+npx wp-create-block example-static --no-wp-scripts
+cd example-static
+
+status "Verifying project..."
+expected=5
+actual=$( find . -maxdepth 1 -type f | wc -l )
+if [ "$expected" -ne "$actual" ]; then
+	error "Expected $expected files in the project root, but found $actual."
+    exit 1
+fi
+expected=6
+actual=$( find src -maxdepth 1 -type f | wc -l )
+if [ "$expected" -ne "$actual" ]; then
+	error "Expected $expected files in the `src` directory, but found $actual."
```

Comment:
@gziolo: In sh and bash, backticks are the old way to represent command substitution. Which means that the line above actually evals as:

```sh
error "Expected $expected files in the $(src) directory ..."
```

which will invoke `src` as a command that it will try to find in `$PATH`, which will either fail or do something that we don't want.

The fix should be to escape:

```sh
error "Expected $expected files in the \`src\` directory, but found $actual."
```

The same applies for this other occurrence in the file:

```sh
error "Expected $expected files in the `build` directory, but found $actual."
```

Reply:
Great to know. I'll update all that tomorrow. Thank you for the comment.

Reply:
It took me 20 days, but PR is ready: https://github.com/WordPress/gutenberg/pull/40473 😅